data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+          type: string
+          description: Title of the book where the annotation was found
+        - name: book_type
+          type: string
+          description: Whether the work is a full book or a section of a book. 
+          constraints:
+            pattern: "Book|Book Section"
+        - name: page
+          type: string
+          description: The page where the annotation was found # QUESTION: What do brackets mean?
+        - name: tags
+          type: string
+          description: Supplemental information about the annotation (e.g. blue ink, arrow)
+        - name: text_content
+          type: any
+          description: Words included in the annotation
+        - name: text_language
+          type: any
+          description: The language used in the annotation
+        - name: text_language_code
+          type: any
+          description: The ISO 639-1 codes for the language used in the annotation
+        - name: text_translation
+          type: any
+          description: 'Translation of the annotation text'
+        - name: quote_content
+          type: string
+          description: The book's text that was highlighted or underlined by the given annotation
+        - name: quote_language
+          type: string
+          description: The language of the book's text that was highlighted by the annotation
+        - name: quote_language_code
+          type: string
+          description: The ISO 639-1 codes for the book's text that was highlighted by the annotation
+        - name: annotator
+          type: string
+          description: The person who made the annotation # QUESTION: Add language about certainty? How did we know this?
+        - name: annotation_region
+          type: string
+          description: A link to the IIIF image of the book's page that has been cropped to show just the given annotation
+          constraints:
+            unique: true
+        - name: page_iiif
+          type: string
+          description: A link to the IIIF image of the book's page
+    scheme: file
+  - encoding: utf-8-sig
+    format: csv
+    hashing: md5
+    name: references
+    path: references.csv
+    description: References to other works made in Jacques Derrida's 1967 book De la grammatologie
+    profile: tabular-data-resource
+    schema:
+      foreignKeys:
+        - fields: book_id
+          reference: 
+            resource: library
+            fields: id
+      fields:
+        - name: id
+          type: string
+          description: An identifier linking to the reference's detail page. Note that these are not unique.
+          constraints:
+            required: true
+        - name: page
+          type: integer
+          description: The page in De la grammatologie where the reference was cited
+          constraints:
+            required: true
+        - name: page_location
+          type: string
+          description: '' # What is this? single letters are the paragraph?
```

Comment:
these are alpha identifiers for the references in the order they occur on the page

Reply:
✅ 15354c07bd6ac9773e7e68a9ea97e99fa6de5735